zipkin-server/src/main/java/zipkin2/server/internal/ZipkinServerConfiguration.java

```diff
@@ -102,18 +94,18 @@
       // and default to a slightly longer timeout on the server to be able to handle these with
       // better error messages where possible.
       sb.requestTimeout(Duration.ofSeconds(11));
-    };
-  }
 
-  @Bean Consumer<MeterRegistry.Config> noAdminMetrics() {
```

Comment:
these consumers created a cycle which we didn't notice before because bytecode magic walked it out. I undid this as at the end of the day the config is mutable anyway cc @anuraaga @trustin